docs/build/build-macOS.md

```diff
@@ -0,0 +1,32 @@
+# Building SRT on macOS
+
+[Homebrew](https://brew.sh/) supports the **srt** formula.
+
+```shell
+brew update
+brew install srt
+```
```

Comment:
Wait a moment.

Why the instruction for Linux is how to build SRT from sources, while for Mac it starts with how to install SRT from the system repository? Maybe both methods should be explained in both cases...